src/templates/base_register.html

```diff
@@ -79,15 +78,14 @@
     })
 });
 </script>
-
-{% endblock %}
-{% block extrajs %}
-{% endblock %}
+{% endblock js %}
```

Comment:
Yes I'm sure, we should write block name. In this file block name is written mostly, Where It was not then I write it. Because   It was little bit confusing that which block is going to start and end.